storage/innobase/log/log0online.c

```diff
@@ -501,9 +501,9 @@ log_online_make_bitmap_name(
 /*=========================*/
 	ib_uint64_t	start_lsn)	/*!< in: the start LSN name part */
 {
-	ut_snprintf(log_bmp_sys->out.name, FN_REFLEN, bmp_file_name_template,
-		    log_bmp_sys->bmp_file_home, bmp_file_name_stem,
-		    log_bmp_sys->out_seq_num, start_lsn);
+	ut_snprintf(log_bmp_sys->out.name, sizeof(log_bmp_sys->out.name), 
```

Comment:
Why the typecast?
Formatting
Add a debug assert that the result string fits into FN_REFLEN

Reply:
It's snprintf, and I'm using sizeof - so it will fit into it.

The warning was generated by the format string, and for that, I can't add a simple assert - I can add the parts of the stringts together manually in the assert, but I'm not sure if that would be a good idea, it's too easy to forget when the pattern is changed.

The cast was just a remainder from a previous different fix for the warning, I removed it.

Reply:
I specifically had in mind an assert that the formatted string < FN_REFLEN, not the bumped size of the buffer, but on second thought such assert is going to be useless anyway